contrib/apelscripts/accountingRun.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# accountingRun.sh
+# sjones@hep.ph.liv.ac.uk, 2019
+# Run the processes of a HTCondor accounting run
+
+/usr/share/apel/condor_ce_blah.sh       # Make the blah file (CE/Security data)
+/usr/share/apel/condor_ce_batch.sh      # Make the batch file (Job times)
+/usr/bin/apelparser                # Read the blah and batch files in
+/usr/bin/apelclient                # Join blah and batch records to make job records
+/usr/bin/ssmsend                   # Send job records into APEL system
```

Comment:
"This" package provides accountingRun.sh <https://github.com/opensciencegrid/htcondor-ce/pull/222#discussion_r275504569>, i.e. htcondor-ce-apelscripts

Reply:
Sorry, by "this", I meant `ssmsend`

Reply:
The ssmsend  tool is provided by the apel-ssm.noarch package. It carries a different version number, since it is not directly related to how the data is created, i.e. ssmsend is more generic.

Reply:
In that case, you can specify this requirement (without the version number) in the spec file with `Requires: apel-ssm`